src/config/metric-names.ts

```diff
@@ -62,3 +62,13 @@ export const metricCreateBranch = {
 	created: `${server}.create-branch.created`,
 	failed: `${server}.create-branch.failed`
 };
+
+export const metricDeploymentPersistent = {
+	toCreate: `${server}.persistent.entry.to-create`,
```

Comment:
btw we don't have anything related to "deployment' in the name of the metric... wondering if that's intentionally?

Reply:
Good catch, I've added some specific name to the metrics itself.